src/routes/transactions/mappers/module-transactions/module-transaction.mapper.ts

```diff
@@ -41,7 +41,7 @@ export class ModuleTransactionMapper {
       txStatus,
       txInfo,
       executionInfo,
-      null, // TODO: include safeAppInfo retrieval logic where needed
```

Comment:
I think this TODO was introduced by mistake, `ModuleTransactions` shouldn't include `SafeAppInfo` so a fixed `null` value is OK.